src/pose.rs

```diff
@@ -0,0 +1,249 @@
+use crate::config::{Color, PoseListenerConfig};
+use crate::transformation::ros_pose_to_isometry;
+use nalgebra::geometry::{Isometry3, Point3};
+use std::option::Option;
+use std::sync::{Arc, RwLock};
+use tui::style;
+use tui::widgets::canvas::Line;
+
+use rosrust;
+
+fn pose_to_arrow(pose: &Isometry3<f64>, length: f64, color: &Color) -> Vec<Line> {
```

Comment:
maybe allow a pose message here? this is isometry3 to arrow. but no strong feelings

Reply:
I can rename them all to iso_to_arrow. Using ros messages to represent data internally is annoying because they have no constructor, no methods etc.